src/services/AsyncStorageService.ts

```diff
@@ -21,6 +21,14 @@ class AsyncStorageService {
     });
   }
 
+  async photosSyncIsEnabled() {
+    return (await this.getItem(AsyncStorageKey.PhotosSyncEnabled)) === 'true' ? true : false;
```

Comment:
return the comparation directly: 
`return (await this.getItem(AsyncStorageKey.PhotosSyncEnabled)) === 'true'`